editor/editor_properties.cpp

```diff
@@ -1856,13 +1857,19 @@ EditorPropertyTransform::EditorPropertyTransform() {
 ////////////// COLOR PICKER //////////////////////
 
 void EditorPropertyColor::_color_changed(const Color &p_color) {
+	if (!Input::get_singleton()->is_mouse_button_pressed(BUTTON_LEFT))
+		emit_changed(get_edited_property(), p_color, "", true);
+	else {
+		CanvasItem* p = dynamic_cast<CanvasItem*>(get_edited_object());
+		if (p)
+			p->set_modulate(p_color);
```

Comment:
There are many places to use.
does it work with `self_modulate` and `Custom colors` of `Label`?

Reply:
Yes and yes

Reply:
hmmm... I'm realizing now that it isn't calling `CanvasItem::set_self_modulate`

Reply:
Fixed